client/src/containers/pages/Home/tests/Home.test.tsx

```diff
@@ -70,4 +75,38 @@ describe('containers/pages/Home', () => {
       username: 'superset-user',
     });
   });
+
+  it('Check if translation is working or not', () => {
+    const { authenticated, user, extraData } = getOpenSRPUserInfo({
+      oAuth2Data: {
+        access_token: 'hunter2',
+        expires_in: '3599',
+        state: 'opensrp',
+        token_type: 'bearer',
+      },
+      preferredName: 'Superset User',
+      roles: ['ROLE_EDIT_KEYCLOAK_USERS'],
+      username: 'superset-user',
+    });
+    store.dispatch(authenticateUser(authenticated, user, extraData));
+
+    act(() => {
+      void i18n.use(initReactI18next).init({
+        resources: Language,
+        lng: 'French',
+        fallbackLng: 'English',
+        interpolation: { escapeValue: false },
+      });
+    });
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedHomeComponent />
+        </Router>
+      </Provider>
+    );
+
+    expect(wrapper.find('h3').text()).not.toEqual('Welcome to OpenSRP');
```

Comment:
@ashfahan Do you think it would be better to test that it's equal to the French equivalent instead?

Reply:
sure 👌